src/pages/api/clinical-trial-search.ts

```diff
@@ -125,8 +125,7 @@ async function callWrappers(matchingServices: string[], query: Bundle, patientZi
   const errors = wrapperResults.filter(result => result.status == 500);
 
   // Combine the responses that were successful
-  const combined: StudyDetailProps[] = [];
-  const uniqueTrialIds = new Set<string>();
+  const uniqueTrials: { [key: string]: StudyDetailProps } = {};
```

Comment:
I'm tempted to say this should really be a `Map<string, StudyDetailProps>`, but this almost certainly doesn't really matter. (The only reason to do that would be if we were likely to get keys that interfered with built-in JavaScript names, and even if we do, it shouldn't really matter.)

Reply:
I don't mind switching it to a `Map`, but the keys should this _should_ only be NCT ids so there shouldn't be any conflict.